examples/asr/speech_to_text_eval.py

```diff
@@ -173,6 +242,30 @@ def main(cfg: EvaluationConfig):
 
     logging.info(f'Dataset WER/CER ' + str(round(100 * wer, 2)) + "%/" + str(round(100 * cer, 2)) + "%")
 
+    if cfg.use_per:
+        logging.info(f'Dataset PER ' + str(round(100 * per, 2)) + '%')
+
+        if HAVE_TABLUATE_AND_PANDAS:
```

Comment:
Put into transcribe utils

Reply:
Probably, it would be clearer to add it as a method to `DatasetPunctuationErrorRate` class. Done it with the commit [3c9bfc5](https://github.com/NVIDIA/NeMo/pull/7538/commits/3c9bfc5191e2a689b2d4ff671bbe5449051058fd)